.clang-format

```diff
@@ -1 +1,7 @@
 BasedOnStyle: Chromium
+AllowShortBlocksOnASingleLine: true
+AllowShortCaseLabelsOnASingleLine: true
+AllowShortFunctionsOnASingleLine: true
+AllowShortIfStatementsOnASingleLine: true
+AllowShortLoopsOnASingleLine: true
+AlwaysBreakTemplateDeclarations: MultiLine
```

Comment:
Perhaps now that we have the opt-opts for specific files like the `-inl.h` we don't actually need/want these?

Reply:
Hmm, yeah, it's ambiguous because it turns out the codebase is not very consistent in one way or the other. Without these options there were many short if/for/functions that were converted to multiple lines, but with these options now we have maybe as many multiple-line if/for/functions that were converted to single lines, so I guess either way we end up with sizable lines of changes. I'll try deleting this.

Reply:
Hmm, you mean reverting all those `Allow***` lines, or only these four lines? Because there are also
```
AllowShortBlocksOnASingleLine: true                                      
AllowShortCaseLabelsOnASingleLine: true
```
above these lines.

Reply:
OK, I removed all added lines from .clang-format except for 
```
AllowShortCaseLabelsOnASingleLine: true
```

Let me know if you want this to be removed as well.